src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -226,7 +230,7 @@ private static String prepareArgv0(Map<String, String> additionalEnvironment) th
                      .map(e -> e.getKey() + "='" + e.getValue() + "'")
                      .collect(joining(" "));
     String command = Stream
-                         .concat(Stream.of(javaBinary().toString()), javaBinaryArgs())
```

Comment:
Why change this?

Reply:
Changing this because Android doesn't actually execute a Java binary, and `javaBinary()` returns a `Path` that is then converted to a string here.

Since we execute a binary named `app_process`, we simply need to return the linux execute command `exec`.